packages/ckeditor5-dev-utils/lib/translations/multiplelanguagetranslationservice.js

```diff
@@ -211,7 +211,15 @@ module.exports = class MultipleLanguageTranslationService extends EventEmitter {
 			const stringifiedTranslations = JSON.stringify( translatedStrings )
 				.replace( /"([a-z]+)":/g, '$1:' );
 
-			const outputBody = `CKEDITOR_TRANSLATIONS.add('${ language }',${ stringifiedTranslations })`;
+			const outputBody = (
+				// We need to ensure that the CKEDITOR_TRANSLATIONS variable exist and if it exists, we need to extend it.
+				'(d=>' +
```

Comment:
I didn't meant why it was a `function`. I meant – why it's minified?

Reply:
Because it won't be transpiled later. It's added to the built asset. And we want the minified code in the production mode.

Reply:
Hmm. Maybe I'm a little bit wrong here. After changing listening to `emit` event to listening to `optimize-chunk-assets` it might be optimized later. I'll check it.

Reply:
Yes. In fact, transpilation is done later, but it breaks sourcemaps. I'll try to work-around this quickly.